pipeline/workflow-builder.Dockerfile

```diff
@@ -16,6 +16,9 @@ RUN ls -la ./resources
 
 ENV swf_home_dir=/home/kogito/serverless-workflow-project
 RUN if [[ -d "./resources/src" ]]; then cp -r ./resources/src/* ./src/; fi
+# Hacking pom.xml to add early access repo
```

Comment:
pls replace this hack with:
```
# Temp hack to provide persistence artifacts
ENV MAVEN_REPO_URL=https://maven.repository.redhat.com/earlyaccess/all
```

Reply:
Done

https://github.com/parodos-dev/serverless-workflows/issues/167 to remove the fix